docs/running_rune_with_occlum.md

```diff
@@ -54,7 +61,7 @@ RUN mkdir -p /run/rune/${OCCLUM_INSTANCE_DIR}
 WORKDIR /run/rune
 
 COPY ${OCCLUM_INSTANCE_DIR} ${OCCLUM_INSTANCE_DIR}
-COPY ${OCCLUM_INSTANCE_DIR}/build/lib/libocclum-pal.so /usr/lib/liberpal-occlum.so
+COPY ${OCCLUM_INSTANCE_DIR}/build/lib/libocclum-pal.so.0.14.0 /usr/lib/libocclum-pal.so.0.14.0
```

Comment:
Don't use the version number. This makes your doc inconsistent. Currently you cannot snip the version number of source operand due to the issue of occlum but at least you can snip the version number in dest operand.